### PR TITLE
Update linter and formatter

### PR DESCRIPTION
**Pull request by RaananW**
Updating linter and formatter to a newer version.

Old formatter seems to have ignored a rule that was set, so a few files require formatting.

THis also add a rule to set the auto-fix for type-adding to be separated and not inline.

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Review by carolhmj — APPROVED** [review]

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14461/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14461/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14461/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14461/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14461/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14461/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14461/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14461/merge#BCU1XR#0

File: .eslintrc [hunk 1]
```diff
@@ -130,7 +130,7 @@
         // "no-duplicate-imports": ["error", { "includeExports": true }],
         // should be error, no?
         "@typescript-eslint/no-unused-vars": ["warn", { "argsIgnorePattern": "^_" }],
-        "@typescript-eslint/consistent-type-imports": ["error", { "disallowTypeAnnotations": false }],
+        "@typescript-eslint/consistent-type-imports": ["error", { "disallowTypeAnnotations": false, "fixStyle": "separate-type-imports" }],
```
Comment (sebavan) [inline .eslintrc:133]: If we do not have it as warning or errors, should you call fix in pre commit tasks ?

Comment (RaananW) [reply .eslintrc:133]: that would take a long time for each commit, AND will not solve the issue if it's already inlined.

**Review by sebavan — APPROVED** [review]

**Comment by bjsplat** [comment]
Reviewer - this PR has made changes to one or more package.json files.

**Pull request merged**